styleguide/source/_patterns/01-atoms/copyright.twig

```diff
@@ -0,0 +1,13 @@
+{# Conditionally checks the current year to automatically update the copyright range. #}
+{% set startYear = 1995 %}
+{% set currentYear = "now"|date("Y") %}
+
+{% if startYear == currentYear %}
+    {% set copyrightYears = startYear %}
+{% else %}
+    {% set copyrightYears = startYear ~ ' - ' ~ currentYear %}
+{% endif %}
+
+<p class="ama__copyright ama__type-smaller">
```

Comment:
The class `ama__type-smaller` should use a double hyphen for the modifer making it `ama__type--smaller`.  Using a `<p>` tag for this is a change and gives extra margins around the copyright. This likely needs to continue being  a `<div>` to retain the proper spacing.

Reply:
I agree that it should be `div` since we don't want to imply spacing at the atom level?